src/components/TokenBottomSheet.tsx

```diff
@@ -239,78 +242,91 @@ function TokenBottomSheet<T extends TokenBalance>({
     filterChipsCarouselRef.current?.scrollTo({ x: 0 })
   }
 
+  const handleMeasureHeader = (event: { nativeEvent: { layout: { height: number } } }) => {
+    setHeaderHeight(event.nativeEvent.layout.height)
```

Comment:
Should we reset the height to 0 after closing?

Reply:
why do we want to set it to 0? `onLayout` will be called when the component changes, so if the header components do change then the token bottom sheet will already update in the background. my understanding is that the token bottom sheet isn't unmounted so setting the height to 0 after close will not guarantee that the height stays at 0 until the next open anyway?